example.py

```diff
@@ -3,46 +3,55 @@
 Test psycopg with CockroachDB.
 """
 
-import time
-import random
 import logging
 import os
+import random
+import time
+import uuid
 from argparse import ArgumentParser, RawTextHelpFormatter
 
 import psycopg2
 from psycopg2.errors import SerializationFailure
+import psycopg2.extras
 
 
 def create_accounts(conn):
-    with conn.cursor() as cur:
+    psycopg2.extras.register_uuid()
+    ids = []
+    id1 = uuid.uuid4()
+    id2 = uuid.uuid4()
+    with conn.cursor(cursor_factory=psycopg2.extras.RealDictCursor) as cur:
         cur.execute(
-            "CREATE TABLE IF NOT EXISTS accounts (id INT PRIMARY KEY, balance INT)"
+            "CREATE TABLE IF NOT EXISTS accounts (id UUID PRIMARY KEY, balance INT)"
         )
         cur.execute(
-            "UPSERT INTO accounts (id, balance) VALUES (1, 1000), (2, 250)")
+            "UPSERT INTO accounts (id, balance) VALUES (%s, 1000), (%s, 250)", (id1, id2))
         logging.debug("create_accounts(): status message: %s",
                       cur.statusmessage)
     conn.commit()
+    ids.append(id1)
+    ids.append(id2)
+    return ids
 
 
 def delete_accounts(conn):
-    with conn.cursor() as cur:
+    with conn.cursor(cursor_factory=psycopg2.extras.RealDictCursor) as cur:
```

Comment:
I don't think you need a dict cursor to run a delete, because there is no record to fetch.

OTOH, if you want to consistently return your records as dictt, you can use `cursor_factory=psycopg2.extras.RealDictCursor` [with the `connect()` call](https://www.psycopg.org/docs/module.html#psycopg2.connect), so that `conn.cursor()` will just serve you a dict cursor without specifying the factory every time.

Reply:
Ok, I set it in `connect`.